redash/query_runner/tinybird.py

```diff
@@ -0,0 +1,126 @@
+import logging
+
+import requests
+
+from redash.query_runner import register
+from redash.query_runner.clickhouse import ClickHouse
+
+logger = logging.getLogger(__name__)
+
+
+class Tinybird(ClickHouse):
```

Comment:
To be clear, we intended on using the `Clickhouse.run_query` implementation? It relies on `Clickhouse._clickhouse_query`, so I want to make sure Tinybird and Clickhouse have the same response format.

It also appears that function logs a line saying `Clickhouse is about to execute query`, which would be inaccurate for any Tinybird logs.

Reply:
Should we wait for the response from @pandomic about this one before merging, just in case it needs changing?

Reply:
I think we're good either way (because we're supposed to be inheriting Clickhouse anyway). But I think it'd be better to have the logs more accurate :)

Reply:
Cool. :smile:

Reply:
👍 Tinybird should have the same JSON response format (at least when it comes to data) according to docs:
https://clickhouse.com/docs/en/interfaces/formats#json
https://www.tinybird.co/docs/api-reference/query-api.html

We have been actually using this implementation for almost two years in production now 🙂